.github/workflows/reusable-release.yml

```diff
@@ -366,11 +366,12 @@ jobs:
           sonatypeBundleDirectory := new File("$LOCAL_ARTIFACTS_STAGING_PATH")
           sonatypeProfileName := "${{ inputs.SONATYPE_PROFILE_NAME }}"
           sonatypeCredentialHost := "${{ inputs.SONATYPE_CREDENTIAL_HOST }}"
+          version := "${{ needs.push-release-commit.outputs.release_version }}"
```

Comment:
The `version` key is pulled into the description that's eventually used by the `sbt-sonatype` plugin

https://github.com/xerial/sbt-sonatype/blob/4e18ee06d8e0cb72d289c6d1cf455cc63e5661dc/src/main/scala/xerial/sbt/sonatype/SonatypeClient.scala#L121